src/main/resources/default/taglib/t/fileField.html.pasta

```diff
@@ -44,15 +44,162 @@
                @if (isFilled(tabIndex)) { tabindex="@tabIndex" }/>
         <div class="input-group-append select-btn">
             <span class="input-group-text">
-                <span class="fa fa-upload"></span>
+                <span class="fa fa-cloud-upload-alt"></span>
             </span>
         </div>
     </div>
+    <div id="sirius-upload-progress-@localId" class="mt-3 mb-3 dropzone-items">
+        <div class="sirius-upload-hover h-100 d-none flex-column align-items-center justify-content-center">
+            <i class="fa fa-2x fa-cloud-upload-alt"></i>
+            <span>@i18n("template.html.uploadArea")</span>
+        </div>
+    </div>
 
     <script type="text/javascript">
         sirius.ready(function () {
+            let _input = document.querySelector('#___localId input');
+            new Dropzone("#sirius-upload-progress-@localId", {
+                url: function (files) {
+                    let value = _input.value;
+
+                    if (value == null || value === '') {
+                        value = '@basePath';
+                    } else {
+                        value = value.substr(0, value.lastIndexOf("/"))
+                    }
+                    return '/fs/upload?filename=' + encodeURIComponent(files[0].name) + '&path='  + value;
+                },
+                sendFileAsBody: true,
+                parallelUploads: 1,
+                maxFilesize: null,
+                acceptedFiles: '___allowedExtensions',
+                previewTemplate: '' +
```

Comment:
would it make sense to extract the JS code out of the template in a fileField.js.pasta or the like (just like the vfs-modals.js)?